lib/validate.js

```diff
@@ -27,13 +19,14 @@ const placeholders = message => (
 );
 
 const validMessage = (message, source) => {
-    const transPlaceholders = placeholders(message);
-    const srcPlaceholders = placeholders(source);
+    const transPlaceholders = placeholders(message.toString());
+    const srcPlaceholders = placeholders(source.toString());
```

Comment:
Should `.toString()` have been here the whole time? If not, what changed? Just curious :)